runtime/src/main/java/org/corfudb/util/NetworkUtils.java

```diff
@@ -0,0 +1,47 @@
+package org.corfudb.util;
+
+import java.net.Inet4Address;
+import java.net.InetAddress;
+import java.net.NetworkInterface;
+import java.net.SocketException;
+import java.util.Enumeration;
+
+import lombok.extern.slf4j.Slf4j;
+
+import org.corfudb.runtime.exceptions.unrecoverable.UnrecoverableCorfuError;
+
+/**
+ * Network utility methods.
+ *
+ * <p>Created by zlokhandwala on 3/6/18.
+ */
+@Slf4j
+public class NetworkUtils {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)